src/runtime_src/driver/xclng/drm/xocl/subdev/xvc.c

```diff
@@ -341,8 +341,7 @@ static int xvc_probe(struct platform_device *pdev)
 	struct resource *res;
 	struct xocl_dev_core *core;
 	int err;
-
-	xvc = xocl_drvinst_alloc(&pdev->dev, sizeof(*xvc));
+	xvc = devm_kzalloc(&pdev->dev, sizeof(*xvc), GFP_KERNEL);
```

Comment:
Is this miss merge?

Reply:
yes, this is a wrong merge. I have reverted `xvc.c` to the version on Xilinx master